packages/react/src/components/Menu/MenuItem.tsx

```diff
@@ -239,11 +238,13 @@ class MenuItem extends AutoControlledComponent<WithAsProp<MenuItemProps>, MenuIt
                 styles: styles.icon,
               },
             })}
-          {rtlTextContainer.createFor({ element: content })}
+          {Box.create(content, {
+            defaultProps: { as: 'span', styles: styles.content },
+          })}
           {menu &&
             Icon.create(indicatorWithDefaults, {
               defaultProps: {
-                name: vertical ? 'stardust-arrow-end' : 'stardust-arrow-down',
+                name: vertical ? 'chevron-right' : 'chevron-down',
```

Comment:
I _think_ the `stardust` prefix for icons is used to be able to swap out icons per theme. So I think you might want to update or add a reference.

https://github.com/stardust-ui/react/blob/master/packages/react/src/themes/teams/index.tsx

Reply:
I'm not sure how I would I go about changing modifying 'stardust-arrow-down' and 'stardust-arrow-end' just in the context of Menu.
![image](https://user-images.githubusercontent.com/15043460/63550700-124cf180-c4e8-11e9-8527-9d7ade395910.png)
Would change it for every case of 'stardust-arrow-down and -right' which isn't what I want - I don't think it is at least.
Doing this would mean new assets are needed for the default Stardust theme right?
![image](https://user-images.githubusercontent.com/15043460/63550786-3d374580-c4e8-11e9-8922-c8ebeeb4cc61.png)